autoload/jedi.vim

```diff
@@ -367,19 +395,47 @@ endfunction
 " ------------------------------------------------------------------------
 
 function! jedi#add_goto_window(len) abort
-    set lazyredraw
-    cclose
     let height = min([a:len, g:jedi#quickfix_window_height])
-    execute 'belowright copen '.height
-    set nolazyredraw
+    " " Q: why the cclose?  To get the height always?
+    " " Should save/restore lazyredraw otherwise.
+    " let save_lazyredraw = &lazyredraw
+    " set lazyredraw
+    " try
+    "     execute 'belowright cwindow '.height
+    " finally
+    "     let &lazyredraw = save_lazyredraw
+    " endtry
+
+    " Using :cwindow allows to stay in the current window in case it is opened
+    " already.
+    execute 'belowright cwindow '.height
```

Comment:
@davidhalter 
Please comment on the code I've commented and are about to remove.

Reply:
I don't really care about the code above. As long as it works - which it does it's fine. Also the reduction of complicated code is always a good thing.